### DISCUSSION TITLE
Option selection in the search bar before will enter, like most upvoted, from least to most, or shuffle. (Web site)

### DISCUSSION THREAD

**Discussion by Laurennn123** [dailydotdev/daily#1169] [Feature Requests]
I came up with the idea of this because recently I wanted to find the most upvoted pertaining to what I searched. This is my first time sharing ideas, and I noticed if it was okay to share.

**Comment by idoshamun** [comment]
Hi @Laurennn123,

Thanks for the suggestion. Can you elaborate your idea please so I'll better understand?

**Reply by Laurennn123** [reply]
Hello,

The idea I was talking about was that after searching for a specific post, it has a selection option from most upvoted post to low, low to most upvoted, recent post, or shuffle.
